### ISSUE TITLE
Improving the algorithm and compare notes with others

### ISSUE THREAD

**Issue by DonaldTsang** [deeppomf/DeepMindBreak#13]
- https://github.com/leehomyc/Faster-High-Res-Neural-Inpainting
- https://github.com/pathak22/context-encoder
- https://github.com/JiahuiYu/generative_inpainting
- https://github.com/satoshiiizuka/siggraph2017_inpainting
- https://github.com/marcbelmont/cnn-watermark-removal
- https://github.com/zavolokas/Inpainting
- https://github.com/BoyuanJiang/context_encoder_pytorch
- https://github.com/akmtn/pytorch-siggraph2017-inpainting

**Comment by deeppomf** [comment]
Thank you for sharing these papers.
DeepMindBreak V2 uses an algorithm that's better than those projects.

**Comment by DonaldTsang** [comment]
@deeppomf are there any data analysis papers that shows this e.g. accuracy/precision/F1, human preference based scoring?

**Comment by Kadantte** [comment]
I think he will post with DMB v2 when he released it

**Comment by DonaldTsang** [comment]
@Kadantte when compared to other Architectures I am sure

**Comment by deeppomf** [comment]
The paper I used for DMB v2 has better qualitative evaluations, PSNR, SSIM, and Inception Score.

**Comment by Kadantte** [comment]
Are u gonna update Models ? , I wan't to help with Models but i don't have a good GPU for that .

**Comment by DonaldTsang** [comment]
@deeppomf in DMBv3 could you expand the tested Neural Network Architecture to include more models?  
Also thanks @Kadantte if you can find more models to test that would be great.

**Comment by DonaldTsang** [comment]
@deeppomf Did you rename the library or is DCP a new iteration?

**Comment by deeppomf** [comment]
Rename to avoid trouble with Google DeepMind. But this is a completely new code base. Everything from the old version is gone.

**Comment by Kadantte** [comment]
@deeppomf Nice work :D , I'll test it right now :v .
